spoti-vote/src/components/Sidebar/UserContainer.jsx

```diff
@@ -1,5 +1,6 @@
 import React, {Component} from 'react';
 import User from './User.jsx';
+//import userIcon from '../../img/abstract-user-flat-1.svg';
```

Comment:
defaultIcon is needed

Reply:
not anymore, the server will provide a link to the default icon if needed